pkg/cmd/server/start/master_args.go

```diff
@@ -71,16 +71,25 @@ func NewDefaultMasterArgs() *MasterArgs {
 		MasterPublicAddr: flagtypes.Addr{Value: "localhost:8443", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: true}.Default(),
 		DNSBindAddr:      flagtypes.Addr{Value: "0.0.0.0:53", DefaultScheme: "http", DefaultPort: 53, AllowPrefix: true}.Default(),
 
+		ConfigDir: &util.StringFlag{},
```

Comment:
Just so everyone remembers, we asked Clayton in person and heard that writing to the current directory in the absence of the config is ok and we don't want to expose a separate `--config-dir` flag because if a specific config file isn't specified, we will modify the content of that config-dir whenever we like.